src/Core/TestInterface.php

```diff
@@ -90,6 +90,13 @@ public function getPending();
     public function setPending($state);
 
     /**
+     * Return whether or not the test is focused
+     *
+     * @return bool
+     */
+    public function getFocused();
```

Comment:
Definitely, but that would also be inconsistent with `getPending()`. In light of that, what do you think I should call it?